modules/_labs/select/react/lib/SelectBase.tsx

```diff
@@ -293,17 +293,27 @@ export default class SelectBase extends React.Component<SelectBaseProps> {
   };
 
   componentDidUpdate(prevProps: SelectBaseProps) {
-    const {focusedOptionIndex, isMenuHidden} = this.props;
+    const {focusedOptionIndex, isMenuHidden, isMenuHiding} = this.props;
 
     // If the menu was just displayed, scroll the focused option into
     // center view
     if (!isMenuHidden && prevProps.isMenuHidden) {
-      this.scrollFocusedOptionIntoView(true);
+      // Delay scrolling by a frame to ensure proper measurements of DOM elements
+      // so we know how far to scroll. Without this delay, sometimes measurements
+      // are correct and sometimes they aren't (may be related to the number of
+      // options and/or the length of their labels) -- add the delay to be safe.
+      requestAnimationFrame(() => {
```

Comment:
This `requestAnimationFrame` call will be removed when we convert `SelectBase` into a function component and use hooks for the scrolling behavior in #759.